packages/files-ui/cypress/tests/file-management-spec.ts

```diff
@@ -540,5 +540,24 @@ describe("File management", () => {
       })
 
     })
+
+    it("can see folder on top of the table and file above folder", () => {
```

Comment:
Should this be this ?
```suggestion
    it("can see folder on top of the table and folder above files", () => {
```